api/templates/design.go

```diff
@@ -920,6 +924,14 @@ var Input = Type("input", func() {
 		Example("input")
 	})
 	Attribute("default", Any, "Default value, match defined type.", func() {
+		Meta(oneof.Meta, json.MustEncodeString([]*openapi.Schema{
+			{Type: openapi.String},
+			{Type: openapi.Integer},
+			{Type: "double"},
```

Comment:
Pre typ `double` z nejakeho dovodu chyba konstanta.

Reply:
Asi to berou jakože payload content type bude vždycky JSON a ten má jen `number`, který nerozlišuje precision. Proč je potřeba tady použít `double`?